src/platforms/react-native/manual-setup/manual-setup.mdx

```diff
@@ -98,6 +98,22 @@ We enable the Gradle integration in your `android/app/build.gradle` file by addi
 apply from: "../../node_modules/@sentry/react-native/sentry.gradle"
 ```
 
+### Enable Sentry AGP
```

Comment:
I believe we should document this with 2 features disabled.

`tracingInstrumentation` disabled (this is only enabled if people start the SDKs manually), we can add a comment 
about that, no need to instrument code if performance is disabled by default.

`autoInstallation` disabled (this can conflict with the version of the SDK, RN already adds the dependency of the Android SDK), if the plugin does the same, I am not sure which one will be picked up leading to troubles, maybe this is not an issue, you can confirm with Android folks.

Reply:
Thank you, this sounds like something we need to check. At least in out sample app I've not run into any issues, but I'll verify it.

Reply:
`tracingInstrumentation` enabled can be mentioned here -> https://docs.sentry.io/platforms/react-native/manual-setup/native-init/ just in case APM is enabled on the native side.